src/pybind/mgr/dashboard/frontend/package.json

```diff
@@ -20,6 +20,7 @@
     "i18n:token": "npx i18ntool config token",
     "test": "jest --watch",
     "test:ci": "jest --clearCache && JEST_SILENT_REPORTER_DOTS=true jest --coverage --reporters jest-silent-reporter",
+    "pree2e": "rm -f cypress/reports/results-*.xml || true",
```

Comment:
Is it meant to be run always before e2e & e2e:ci?
Should it be included as the 1st step inside aforementioned scripts?

Reply:
`pre*` and `post*` scripts are [automatically run by npm](https://docs.npmjs.com/misc/scripts).

Reply:
`pree2e` will only be run with `e2e`.
`e2e:ci` will not include it.

Reply:
Fixed... However, I was thinking that given that the first step CI's perform is to wipe-out workspaces (at least our Jenkins does that), we could assume that it's already clean. Nevertheless, this will fully make sure.

Reply:
Both these scripts are only used in local environments.
Better to clean it up otherwise we will get a report folder with thousands of xml files :)